tasks/main.yml

```diff
@@ -1 +1,44 @@
 # role tasks
+
+- name: Find the current locale setting
+  slurp:
+    src: /etc/locale.conf
+  register: host_locale
+
+- name: Update locale if needed
+  become: "{{ localectl_become }}"
+  become_user: "{{ localectl_become_user }}"
+  command: localectl set-locale LANG="{{ localectl_locale }}"
+  when: localectl_locale not in host_locale.content | b64decode
+
+# Grep for KEYMAP= and fail if return code empty
+- name: Find the current keymap setting
+  become: "{{ localectl_become }}"
+  become_user: "{{ localectl_become_user }}"
+  shell: grep /etc/vconsole.conf -v -e '^#' | sed -E -e 's/#.*$//' | grep
```

Comment:
The comment above this task doesn't tell the whole story; this does more than just "Grep for KEYMAP=...". The comment should be updated to reflect the work being done and the intent of this task.

Reply:
Comment has been updated